### DISCUSSION TITLE
¿Cuál es la diferencia principal entre "Merge" y "Rebase" al gestionar un Pull Request en GitHub?

### DISCUSSION THREAD

**Discussion by 4lexxe** [community/community#68282] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

me dieron tarea para la uni: ¿Cuál es la diferencia principal entre "Merge" y "Rebase" al gestionar un Pull Request en GitHub?

**Comment by HyNuwa** [comment]
Mmm fijate en la documentacion de github o pregunta a linus torvalds, no se soy nuevo xd

**Comment by queenofcorgis** [comment, marked as answer]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
